sklearn/decomposition/sparse_pca.py

```diff
@@ -109,6 +109,9 @@ def fit(self, X, y=None):
 
         y : Ignored
 
+        variance : bool, optional
+            If true, compute the explained variance
+
```

Comment:
remove, see previous

Reply:
moved this over to `__init__` (for both)